connector/connect/README.md

```diff
@@ -90,7 +90,7 @@ To use the release version of Spark Connect:
 
 ### Generate proto generated files for the Python client
 1. Install `buf version 1.11.0`: https://docs.buf.build/installation
-2. Run `pip install grpcio==1.48.1 protobuf==3.19.5 mypy-protobuf==3.3.0`
+2. Run `pip install grpcio==1.48.1 protobuf==3.19.5 mypy-protobuf==3.3.0 googleapis-common-protos==1.56.4 grpcio-status==1.48.1`
```

Comment:
Maybe would have to add it to https://github.com/apache/spark/blob/master/python/setup.py#L281-L284 and https://github.com/apache/spark/blob/master/python/pyspark/testing/connectutils.py#L22-L38 so the builds without the dependencies at Jenkins pass (https://apache-spark.s3.fr-par.scw.cloud/index.html)